Remove template import scripts and update dev docs (#160)
redhat-appstudio/tssc-sample-templates: README.md

# RHTAP Sample Backstage Templates

## Software Supply Chain Security

This repository provides a suite of language/framework examples that implement end-to-end software supply chain security (SSCS).

> The examples contain an advanced continuous integration pipeline covering building, CVE scanning, security scanning, signatures,  attestations, SLSA provenance and SBOM along with Gitops-based continuous deployment

Reviewer: ```suggestion
The examples contain an advanced continuous integration pipeline covering building, CVE scanning, security scanning, signatures, attestations, SLSA provenance and SBOM along with Gitops-based continuous deployment.
```

> Software Template authors and IDP builders can take these samples to create their own SSCS based starters. The components in the samples are designed to be maintainable to ensure that day 2 maintainance of default build pipelines and deployment yaml ensures teams have

Reviewer: ```suggestion
Software Template authors and IDP builders can take these samples to create their own SSCS based starters. The components in the samples are designed to be maintainable to ensure 2 day maintenance of default build pipelines and deployment yaml ensures teams have.
```

> deployment yaml ensures teams have

@lcarva Does this state the deployment yaml ensures teams have something? Or should it be ensures teams have the deployment yaml they need?

Author: This whole sentence is a mess. I'm just gonna drop it.

## Usage in Red Hat Developer Hub

> To import these templates, register a component in on the golden path template page

Reviewer: ```suggestion
To import these templates, register a component in on the golden path template page:
```

![Screenshot](register.png)

Import `https://github.com/redhat-appstudio/tssc-sample-templates/blob/main/all.yaml`

![Screenshot](register2.png)

## Usage in backstage

> Add the following to your `app-config.yaml` file in your backstage configuration

Reviewer: ```suggestion
Add the following to your `app-config.yaml` file in your backstage configuration:
```

![Screenshot](backstage.png)

## On-prem host support

> You need to ensure the on-prem host is being configured in `app-config.yaml` file in your backstage/RHDH configuration

Reviewer: ```suggestion
You need to ensure the on-prem host is being configured in `app-config.yaml` file in your backstage/RHDH configuration.
```

> On `Deployment information` Page, provide your Quay host in the `Image Registry`

Reviewer: ```suggestion
On `Deployment information` Page, provide your Quay host in the `Image Registry`:
```
